masterlist.yaml

```diff
@@ -5647,11 +5663,11 @@ plugins:
           - 'Holidays'
           - '[JK''s Skyrim SE Holidays Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
         condition: 'active("Holidays.esp") and not active("JKs Skyrim_Holidays_Patch.esp") and not active("JKs Skyrim_Dawn of Skyrim_Patch.esp") and not active("Blues Skyrim.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Leaf Rest'
-          - '[JK''s Skyrim SE Leaf Rest Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
-        condition: 'active("LeafRest OnePiece.esp") and not active("JKs Skryim Leafrest Patch.esp") and not active("JKs Skyrim Leafrest Patch.esp")'
+      # - <<: *hasPluginPatch
+      #   subs:
+      #     - 'Leaf Rest'
+      #     - '[JK''s Skyrim SE Leaf Rest Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+      #   condition: 'active("LeafRest OnePiece.esp") and not active("JKs Skryim Leafrest Patch.esp") and not active("JKs Skyrim Leafrest Patch.esp")'
```

Comment:
Patch is currently unavailable. Commented out in case it is restored. 
It may also be possible that it's no longer required.